node/sc/bridge_accounts.go

```diff
@@ -229,11 +236,21 @@ func (acc *accountInfo) LockAccount() error {
 }
 
 // UnLockAccount can unlock the account keystore.
-func (acc *accountInfo) UnLockAccount(passphrase string) error {
+func (acc *accountInfo) UnLockAccount(duration *uint64, passphrase string) error {
```

Comment:
```suggestion
func (acc *accountInfo) UnLockAccount(passphrase string, duration *uint64) error {
```
How about putting optional value last?

Reply:
Right. I will modify this. :)